app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -594,43 +595,54 @@ class SamplesHeatmapView extends React.Component {
           taxonFilterState={this.state.taxonFilterState}
           thresholdFilters={this.state.selectedOptions.thresholdFilters}
           sampleSortType={this.state.selectedOptions.sampleSortType}
+          fullScreen={this.state.hideFilters}
         />
       </ErrorBoundary>
     );
   }
 
+  toggleDisplayFilters = () => {
+    this.setState(prevState => ({ hideFilters: !prevState.hideFilters }));
+  };
+
   render() {
     return (
       <div className={cs.heatmap}>
-        <NarrowContainer>
-          <SamplesHeatmapHeader
-            sampleIds={this.state.sampleIds}
-            data={this.state.data}
-            onDownloadSvg={this.handleDownloadSvg}
-            onDownloadPng={this.handleDownloadPng}
-            onDownloadCsv={this.handleDownloadCsv}
-            onShareClick={this.handleShareClick}
-            onSaveClick={this.handleSaveClick}
+        {!this.state.hideFilters && (
+          <div>
+            <NarrowContainer>
+              <SamplesHeatmapHeader
+                sampleIds={this.state.sampleIds}
+                data={this.state.data}
+                onDownloadSvg={this.handleDownloadSvg}
+                onDownloadPng={this.handleDownloadPng}
+                onDownloadCsv={this.handleDownloadCsv}
+                onShareClick={this.handleShareClick}
+                onSaveClick={this.handleSaveClick}
+              />
+            </NarrowContainer>
+            <NarrowContainer>
+              <SamplesHeatmapControls
+                options={this.getControlOptions()}
+                selectedOptions={this.state.selectedOptions}
+                onSelectedOptionsChange={this.handleSelectedOptionsChange}
+                loading={this.state.loading}
+                data={this.state.data}
+              />
+            </NarrowContainer>
+          </div>
+        )}
+        {this.state.hideFilters && <Divider style="shadow" />}
```

Comment:
I wouldn't use the `<Divider>` component in this case. It doesn't feel like this is a "divider". It's not visually separating two sections of content. Instead, it's hinting that a panel can be opened. We generally try not to add new types or "styles" of a component without good reason.

For now, I would just make this a div. You might nest it under the `filterToggleContainer` div since it's technically part of that component (you would then have to move the onClick to a child)